src/migrations/1729857146775-item-category-to-item.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migrations1729857146775 implements MigrationInterface {
+  name = 'language-category-to-item-1729857146775';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // we ignore english category because default item lang is already english
+    for (const [lang, key] of [
+      ['french', 'fr'],
+      ['german', 'de'],
+      ['arabic', 'ar'],
+      ['italian', 'it'],
+    ]) {
```

Comment:
The order of the loop will matter right ? If an item has both french and italian for example, it will receive italian as the final language ?